### DISCUSSION TITLE
"Classic Explorer navigation bar" No Navigation Bar on Explorer open

### DISCUSSION THREAD

**Discussion by genericadm** [ramensoftware/windhawk-mods#1385] [General]
Hello @m417z 

I'm using "Classic Explorer navigation bar" and have an odd behaviour. When I open Documents from a pin on the taskbar, the window opens without an address bar.
![image](https://github.com/user-attachments/assets/57a42ccb-17a5-4e3c-b9b7-67b889f4f483)
![image](https://github.com/user-attachments/assets/34d80dce-8f78-407a-9e09-3c3ad0f55be9)
If I open a new explorer window from there (CTRL+N), the new window has the address bar.

Any idea what's up?

**Comment by genericadm** [comment]
@m417z NEVERMIND! I think it was an issue with the item I had pinned. I unpinned it and then pinned a new one and now the issue is resolved. :)

**Comment by genericadm** [comment]
Closing this discussion.

**Comment by m417z** [comment]
Looks like this issue: https://github.com/ramensoftware/windhawk-mods/issues/487.
This workaround might work: https://github.com/ramensoftware/windhawk-mods/issues/487#issuecomment-1940583008.
